interface/main/about_page.php

```diff
@@ -33,12 +33,23 @@
 ?>
  <html>
   <head>
-  <link rel="stylesheet" href="<?php echo $css_header;?>" type="text/css">
+  <link rel=stylesheet href="<?php echo $css_header;?>" type="text/css">
+  <link rel=stylesheet href="../themes/login.css" type="text/css">
+  <link rel="stylesheet" href="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-ui-1-11-4/themes/base/jquery-ui.min.css" />
+  <link rel="shortcut icon" href="<?php echo $webroot; ?>/interface/pic/favicon.ico" />
+
+  <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-min-2-2-0/index.js"></script>
+  <script type="text/javascript" src="<?php echo $GLOBALS['webroot'] ?>/public/assets/jquery-ui-1-11-4/jquery-ui.min.js"></script>
```

Comment:
Should be using the $GLOBALS['assets_static_relative'] rather than echo $GLOBALS['webroot'] . This allows us to move the $GLOBALS['assets_static_relative'] path as needed in the future.